src/containers/classrooms/ClassroomsTableContainer.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Actions } from 'jumpstate';
+import AstroClassroomsTable from '../../components/astro/AstroClassroomsTable';
+
+import {
+  CLASSROOMS_INITIAL_STATE, CLASSROOMS_PROPTYPES
+} from '../../ducks/classrooms';
+import {
+  ASSIGNMENTS_INITIAL_STATE, ASSIGNMENTS_PROPTYPES
+} from '../../ducks/assignments';
+
+class ClassroomsTableContainer extends React.Component {
+  selectClassroom(classroom) {
+    Actions.classrooms.selectClassroom(classroom);
+  }
+
+  render() {
+    if (this.props.selectedProgram && this.props.selectedProgram.slug === '/astro-101-with-galaxy-zoo') {
```

Comment:
This `.slug` makes me wonder if there's a better way we can organise our Projects; perhaps we need to consider organising Program-specific variables into a Program.config file, then place all those Program-specific config and components files into their own module-folder?

_(This is a minor comment for future discussion, and does not affect the review of this PR.)_

Reply:
e.g. at the moment the `.slug` value (`/astro-101-with-galaxy-zoo`) exists as a (temporary?) const/config in `src/ducks/programs.js`, so that's a prime candidate for putting that into a `/lib/i2a/project.config` file so we can ensure consistent values across all components used by the Program.

Reply:
EDIT: I just properly digested the new updates to `src/ducks/programs.js` - the Duck IS the Program config file now! Sorry, missed that.

In any case, that means I need to change my comment to: we really should start using the config values we've set up instead of hard-coding string values. This remains a minor comment, though.